src/figuresClasses.ts

```diff
@@ -1,19 +1,101 @@
+const ERROR_MESSAGE = {
+  wrongLength: 'length is <= 0',
+  tooLongSide: 'the longest side of a triangle is >= than a sum of two others',
+};
+
+const roundToTwoDecimals = (value: number): number => {
+  return Math.floor(value * 100) / 100;
+};
```

Comment:
If you want to round a number to two decimal places, you can use the `toFixed(2)` method:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed

Reply:
i try this method at first, but result was little different then i need